circuit_maintenance_parser/output.py

```diff
@@ -197,7 +197,7 @@ def validate_empty_strings(cls, value):
     def validate_empty_circuits(cls, value, values):
         """Validate non-cancel notifications have a populated circuit list."""
         values = values.data
-        if len(value) < 1 and str(values["status"]) in ("CANCELLED", "COMPLETED"):
+        if len(value) < 1 and str(values["status"]) not in ("Status.CANCELLED", "Status.COMPLETED"):
```

Comment:
what if you just compare the enum value directly?
```suggestion
        if len(value) < 1 and values["status"] not in (Status.CANCELLED, Status.COMPLETE):
```